### PR TITLE
link and version

### PR DESCRIPTION
**Pull request by jdroenner**
- [x] I agree to follow the project's [code of conduct](https://github.com/georust/gdal/blob/master/CODE_OF_CONDUCT.md).
- [x] I added an entry to `CHANGES.md` if knowledge of this change could be valuable to users.
---

### CODE REVIEW

**Review by lnicola — APPROVED** [review]
I wonder if we shouldn't emit something like `9090900` in the docs.rs case, so everything gets enabled.

File: build.rs [hunk 1]
```diff
@@ -1,11 +1,7 @@
-use std::str::FromStr;
-
-fn gdal_version_info() -> String {
-    gdal_sys::gdal_version_docs_rs_wrapper()
-}
+use std::{env, str::FromStr};
 
 fn main() {
-    let gdal_version_string = gdal_version_info();
+    let gdal_version_string = env::var("DEP_GDAL_VERSION_NUMBER").unwrap();
```
Comment (metasim) [inline build.rs:4]: Though very unlikely, consider using `.expect(...)` with a helpful message to indicate it was expected that `gdal-sys/build.rs` would have emitted `cargo:version-number`.

File: gdal-sys/build.rs [hunk 2]
```diff
@@ -59,45 +59,24 @@ fn find_gdal_dll(lib_dir: &Path) -> io::Result<Option<String>> {
...
-                let res_ptr = crate::GDALVersionInfo(c_key.as_ptr());
-                let c_res = std::ffi::CStr::from_ptr(res_ptr);
-                c_res.to_string_lossy().into_owned()
-            }
-        }"#
-        .to_string()
-    };
-    std::fs::write(out_path, docs_helper_code.as_bytes()).unwrap();
-}
-
```
Comment (metasim) [inline gdal-sys/build.rs:87]: ❤️

File: gdal-sys/src/lib.rs [hunk 3]
```diff
@@ -5,4 +5,3 @@
 #![allow(rustdoc::bare_urls)]
 
 include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
-include!(concat!(env!("OUT_DIR"), "/docs_rs_helper.rs"));
```
Comment (metasim) [inline gdal-sys/src/lib.rs:8]: This must have been satisfying to eliminate! 😄

**Review by metasim — APPROVED** [review]

File: build.rs (hunk 1 above)
Comment (jdroenner) [reply build.rs]: [4e263c9](https://github.com/georust/gdal/pull/377/commits/4e263c94e97ca2cf1f2d468766125e9531178012)

File: gdal-sys/src/lib.rs (hunk 3 above)
Comment (jdroenner) [reply gdal-sys/src/lib.rs:8]: yes :D

**Pull request merged**